### PR TITLE
feat(ml): ViT support for Realformer

### PR DESCRIPTION
**Pull request by beniz**
From https://arxiv.org/abs/201211747v2. Will provide some accuracy metrics on simple image classification datasets for ViT tiny.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -743,6 +743,7 @@ embedding_size  | int    | yes      | 768     | embedding size for NLP models
 freeze_traced   | bool   | yes      | false   | Freeze the traced part of the net during finetuning (e.g. for classification)
 retain_graph	| bool	 | yes	    | false   | Whether to use `retain_graph` with torch autograd
 template        | string | yes      | ""      | for language models, either "bert" or "gpt2", "recurrent" for LSTM-like models (including autoencoder), "nbeats" for nbeats model
+template        | string | yes      | ""      | for language models, either "bert" or "gpt2", "recurrent" for LSTM-like models (including autoencoder), "nbeats" for nbeats model, "vit" for vision transformer
```
Comment (BynaryCobweb) [inline docs/api.md:746]: duplicate line

Comment (beniz) [reply docs/api.md:745]: fixed, thanks!

**Review by BynaryCobweb — APPROVED** [review]

**Comment by beniz** [comment]
@mergifyio update

**Comment by sileht** [comment]
@Mergifyio rebase

**Pull request merged**